internal/blueprint/customizations.go

```diff
@@ -187,3 +194,10 @@ func (c *Customizations) GetServices() *ServicesCustomization {
 
 	return c.Services
 }
+
+func (c *Customizations) GetFilesystem() []FilesystemCustomization {
```

Comment:
Maybe `GetFilesystems` would be better since the method returns a slice?

Reply:
Good point!